src/Mvc/Mvc.ViewFeatures/ref/Microsoft.AspNetCore.Mvc.ViewFeatures.netcoreapp3.0.cs

```diff
@@ -610,6 +610,12 @@ public partial class MvcForm : System.IDisposable
         public void EndForm() { }
         protected virtual void GenerateEndForm() { }
     }
+    public enum RenderMode
+    {
+        Static = 1,
```

Comment:
Start at 0? Why not?

Reply:
I don't want to allow `default` being passed to the method calls.